### PR TITLE
PE-1465: feat(localization ja): adds translations for japanese

### PR DESCRIPTION
**Pull request by matibat**
Changes

- Adds the ARB file for Japanese

### CODE REVIEW

**Comment by matibat** [comment]
There's an issue with the translations of `aggreeToTerms`: the text doesn't match

File: lib/l10n/app_ja.arb [hunk 1]
```diff
@@ -0,0 +1,1361 @@
...
+  },
+  "addProfileEmphasized": "プロフィールを追加する",
+  "@addProfileEmphasized": {
+    "description": "Add a wallet. Emphasized with upper case"
+  },
+  "addSomeFiles": "ここには何もありません ファイルを追加するには [新規作成]をクリックしてください",
+  "@addSomeFiles": {
+    "description": "Empty folder"
+  },
+  "aggreeToTerms_body": "ArDriveの利用規約とプライバシーポリシーに同意する",
```
Comment (matibat) [inline lib/l10n/app_ja.arb:15]: This wasn't mentioning "terms of service"

Screen Shot 2022-08-31 at 12 43 22


```suggestion
  "aggreeToTerms_body": "ArDriveのサービス利用規約とプライバシーポリシー同意する",
```

**TODO** we must update this text in the translation platform as well

File: lib/l10n/app_ja.arb [hunk 2]
```diff
@@ -0,0 +1,1361 @@
...
+  "@@locale": "ja",
+  "addnewManifestEmphasized": "新規マニフェストを追加する",
+  "@addnewManifestEmphasized": {
+    "description": "There is a non-manifest entity with the same name"
+  },
+  "addProfileEmphasized": "プロフィールを追加する",
+  "@addProfileEmphasized": {
+    "description": "Add a wallet. Emphasized with upper case"
+  },
+  "addSomeFiles": "ここには何もありません ファイルを追加するには [新規作成]をクリックしてください",
```
Comment (matibat) [inline lib/l10n/app_ja.arb:11]: Japanese quotation marks

```suggestion
  "addSomeFiles": "ここには何もありません ファイルを追加するには「新規作成」をクリックしてください",
```

File: lib/l10n/app_ja.arb [hunk 3]
```diff
@@ -0,0 +1,1361 @@
...
+  },
+  "nextEmphasized": "次へ",
+  "@nextEmphasized": {
+    "description": "Next step"
+  },
+  "noDeleteButton": "重要：削除ボタンはありません（あなたにも私たちにも）！",
+  "@noDeleteButton": {
+    "description": "Uploaded items cannot be deleted"
+  },
+  "noDrives": "個人用または接続されたドライブがありません。[新規作成]ボタンをクリックして,\n追加してください！\n",
```
Comment (matibat) [inline lib/l10n/app_ja.arb:891]: Japanese quotation marks

```suggestion
  "noDrives": "個人用または接続されたドライブがありません。「新規作成」ボタンをクリックして,\n追加してください！\n",
```

**Review by matibat — COMMENTED** [review]
Issues with quotation marks - **TODO** update the text in the translation platform as well

**Review by karlprieb — DISMISSED** [review]

File: lib/l10n/app_ja.arb [hunk 4]
```diff
@@ -0,0 +1,1361 @@
...
+  },
+  "nextEmphasized": "次へ",
+  "@nextEmphasized": {
+    "description": "Next step"
+  },
+  "noDeleteButton": "重要：削除ボタンはありません（あなたにも私たちにも）！",
+  "@noDeleteButton": {
+    "description": "Uploaded items cannot be deleted"
+  },
+  "noDrives": "個人用または接続されたドライブがありません。「新規作成」ボタンをクリックして,\n追加してください！\n",
```
Comment (matibat) [inline lib/l10n/app_ja.arb:891]: That line break at the end is not in the original translation
```suggestion
  "noDrives": "個人用または接続されたドライブがありません。「新規作成」ボタンをクリックして,\n追加してください！",
```

File: lib/l10n/app_ja.arb [hunk 5]
```diff
@@ -0,0 +1,1361 @@
...
+  "@driveWasRenamed": {
+    "description": "Drive activity (journal) entry: renamed",
+    "placeholders": {
+      "driveName": {
+        "type": "String",
+        "example": "drive name"
+      }
+    }
+  },
+  "duplicateFiles": "{numberOfDuplicateFiles,plural, =1{重複する１個のファイルが見つかりました} =other{重複する複数のファイルが見つかりました}}\"",
```
Comment (matibat) [inline lib/l10n/app_ja.arb:378]: That extra escaped double-quote breaks plurals
```suggestion
  "duplicateFiles": "{numberOfDuplicateFiles,plural, =1{重複する１個のファイルが見つかりました} =other{重複する複数のファイルが見つかりました}}",
```

**Review by matibat — COMMENTED** [review]
Some more issues with the translations

**Comment by matibat** [comment]
Updated all translations on Localizely:
- noDrives
- duplicateFiles
- aggreeToTerms_body
- addSomeFiles

**Review by karlprieb — APPROVED** [review]

**Pull request merged**